modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/ImageField.testcase

```diff
@@ -77,33 +77,6 @@ definition {
 		AssertElementNotPresent(locator1 = "WCD#WEB_CONTENT_CONTENT_IMAGE");
 	}
 
-	@description = "This is a test for LPS-98073. As a Developer, I want to Implement the image field type"
-	@ignore = "true"
-	@priority = "5"
-	test DragAndDropImageFieldToBuilder {
```

Comment:
Hi @DiegoDOC , I'll choose to keep this test in order to guarantee specific validation for this scenario.
Thanks!